doc/dev_guide/troubleshoot_gcp_wif_sts.md

```
@@ -0,0 +1,61 @@
+# Troubleshoot GCP WIF (STS) Cluster
+
+Common errors during GCP Workload Identity Federation (WIF) installations and how to investigate and fix them.
+
+Open the operator logs:
+
+```bash
+kubectl logs <operator-pod> -n <your-namespace> -c noobaa-operator -f
+```
+
+### Main issues:
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)